utilities/src/main/java/io/syndesis/qe/resource/impl/Jaeger.java

```diff
@@ -112,4 +113,20 @@ private void processResources() {
             }
         }
     }
+
+    //Productised builds need to link syndesis-pull secret and redeploy pods
+    public void ensureImagePull() {
+        OpenShiftUtils.getInstance().serviceAccounts().list().getItems().stream().filter(sa -> sa.getMetadata().getName().contains("jaeger"))
+            .forEach(sa -> {
+                sa.getImagePullSecrets().add(new LocalObjectReference(TestConfiguration.syndesisPullSecretName()));
+                OpenShiftUtils.getInstance().serviceAccounts().createOrReplace(sa);
+            });
+        if (OpenShiftWaitUtils.isPodReady(OpenShiftUtils.getAnyPod("name", "jaeger-operator"))) {
+            OpenShiftUtils.getAnyPod("syndesis.io/component", "syndesis-jaeger")
```

Comment:
Here the pod doesn't contain that label. (it will be added after while cycle in the `jaegerWorkarounds` method)